internal/runbits/rationalize/types.go

```diff
@@ -0,0 +1,9 @@
+package rationalize
+
+// Inner is just an alias because otherwise external use of this struct would not be able to construct the error
+// property, and we want to keep the boilerplate minimal.
+type Inner error
+
+type ErrNoProject struct {
+	Inner
+}
```

Comment:
Having centralized errors in a package called rationalize is a bit confusing. Is this because the error is used by the rationalized functions?

Reply:
Rationalize is the term I want to use for this, so when you read "Rationalize" your mind should immediately make the connection.

"Centralized errors" isn't a term we ever used for this, it's just a generic way of thinking about this problem space. I want to be explicit about it.

Btw happy to explore other terms if you don't like "Rationalize". The intend is that it's both sensible and explicit.

Reply:
I'd prefer not to get caught up in the naming. This makes enough sense to me.